### ISSUE TITLE
same intent gets forwarded if the app is launched from the recent tabs

### ISSUE THREAD

**Issue by vikrantyadav7** [llfbandit/app_links#72]
When the app is opened through a link it passes the intent which is good, but if go back to recent tabs and open the app again it passes the same intent again.

So is there any way we can clear that  intent after its used or we check if the app is being launched through the recent tabs.

I found a stackoverflow question for the same. Can we apply it to our package.
(https://stackoverflow.com/questions/34311501/how-to-clear-intent-data-in-activity-after-open-from-url)

[ x] App Links (Android)  
[ x] Deep Links (Android)

**Comment by llfbandit** [comment]
Thanks for the report and the provided hint to prevent this.

**Comment by llfbandit** [comment]
Available in v3.4.5. I didn't reproduced the issue from the simulator.
So feedback welcome!

**Comment by Ozak93** [comment]
> Available in v3.4.5. I didn't reproduced the issue from the simulator. So feedback welcome!

i upgraded to v3.4.5 and it's still happening in the simulator, if i just hot restart the app after receiving an applink intent then it will always trigger my stream listening to applinks.

**Comment by llfbandit** [comment]
Hot restarting isn't related to this issue. By the way your users can't hot reload and there's nothing to fix in this area.

**Comment by vikrantyadav7** [comment]
Hello @llfbandit,

I've upgraded the package and tested it on both a real device and an emulator. Unfortunately, the result remains the same. The same URL is received from both methods, namely, getInitialLink and getLatestLink.

However, during testing, I came across an interesting log. I'm using a different package called AppsFlyer, and its log contains the following message: "I/AppsFlyer_6.12.2(10912): onBecameForeground I/AppsFlyer_6.12.2(10912): skipping re-use of previously consumed deep link."

It seems to be doing what we're trying to achieve. If you'd like, I can investigate this further and try to figure out a solution.
